xml/admin_operating_pools.xml

```diff
@@ -64,825 +65,827 @@
   To organize data into pools, you can list, create, and remove pools. You can
   also view the usage statistics for each pool.
  </para>
- <sect1 xml:id="ceph-pools-associate">
-  <title>Associating pools with an application</title>
 
+ <sect1 xml:id="ceph-pools-operate-add-pool">
+  <title>Creating a pool</title>
   <para>
-   Before using pools, you need to associate them with an application. Pools
-   that will be used with &cephfs;, or pools that are automatically created by
-   &ogw; are automatically associated.
+   A pool can be created as either <literal>replicated</literal> to recover from lost OSDs by
+   keeping multiple copies of the objects or <literal>erasure</literal> to have
+   generalized RAID5 or 6 capability. Replicated pools require more raw storage,
+   while erasure coded pools require less raw storage. The default setting is
+   <literal>replicated</literal>. For more information on erasure coded pools, see
+   <xref linkend="cha-ceph-erasure"/>.
   </para>
-
   <para>
-   For other cases, you can manually associate a free-form application name
-   with a pool:
+   To create a replicated pool, execute:
   </para>
-
-<screen>&prompt.cephuser;ceph osd pool application enable <replaceable>pool_name</replaceable> <replaceable>application_name</replaceable></screen>
-
-  <tip>
-   <title>Default application names</title>
-   <para>
-    &cephfs; uses the application name <literal>cephfs</literal>, &rbd; uses
-    <literal>rbd</literal>, and &ogw; uses <literal>rgw</literal>.
-   </para>
-  </tip>
-
+<screen>&prompt.cephuser;ceph osd pool create <replaceable>POOL_NAME</replaceable> <replaceable>PG_NUM</replaceable> <replaceable>PGP_NUM</replaceable> replicated <replaceable>CRUSH_RULESET_NAME</replaceable> \
+<replaceable>EXPECTED_NUM_OBJECTS</replaceable></screen>
   <para>
-   A pool can be associated with multiple applications, and each application
-   can have its own metadata. You can display the application metadata for a
-   given pool using the following command:
+   To create an erasure coded pool, execute:
   </para>
-
-<screen>&prompt.cephuser;ceph osd pool application get <replaceable>pool_name</replaceable></screen>
- </sect1>
- <sect1 xml:id="ceph-pools-operate">
-  <title>Operating pools</title>
-
+<screen>&prompt.cephuser;ceph osd pool create <replaceable>POOL_NAME</replaceable></screen>
+  <note>
+    <para>
+      The autoscaler will take care of the remaining optional arguments. For
+      more information, see <xref linkend="op-pgs-autoscaler"/>.
+    </para>
+  </note>
```

Comment:
We got our example commands mixed up here. The command for an erasure coded pool is the minimal command for a replicated pool, whereas the EC pool needs the full argument set (s/replicated/erasure/ of course).

Reply:
The `erasure` variant can also skip most arguments (like pg_num and such), which we should use here. If we have a chapter on erasure coded pools we should link to it in order to explain EC profiles and such.

Reply:
I'll generate a PDF and attach it here, so you can check it out. This is the command for simply creating a pool. This is not the EC pool command.

Reply:
[book-storage-admin_color_en.pdf](https://github.com/SUSE/doc-ses/files/5730185/book-storage-admin_color_en.pdf)

Reply:
I see what's happened, one second.